### PR TITLE
green and red in hours positive or negative

### PR DESCRIPTION
**Pull request by sseerrggii**
Green or red on movements and balance depending on whether positive or negative, so it is easier to see if they are inputs or outputs of hours

### CODE REVIEW

File: app/views/shared/_movements.html.haml [hunk 1]
```diff
@@ -38,6 +38,9 @@
                 - op = { method: :put, data: { confirm: t(".delete_reason") } }
                 = link_to(delete_reason_transfer_path(mv.transfer), op) do
                   %span.glyphicon.glyphicon-trash
-            %td= seconds_to_hm(mv.amount)
+            %td
+              - positive = mv.amount.positive?
+              %strong{ :class => (positive ? "text-success" : "text-danger") }
```
Comment (enricostano) [inline app/views/shared/_movements.html.haml]: Creo que puedes directamente poner:
```
%strong{ :class => mv.amount.positive? ? "text-success" : "text-danger" }
```
y ahorrarte la asignacion de la variable `positive`

File: app/views/users/show.html.haml [hunk 2]
```diff
@@ -103,8 +103,9 @@
             = t(".user_no")
           = @member.member_uid || mdash
           %br
-          %strong
+          - positive = @member.account.try(:balance).positive?
+          %strong.lead{ :class => (positive ? "text-success" : "text-danger") }
```
Comment (enricostano) [inline app/views/users/show.html.haml]: lo mismo

File: app/views/shared/_movements.html.haml (hunk 1 above)
Comment (sseerrggii) [inline app/views/shared/_movements.html.haml]: Creo que si no se pasa de tamaño de linea

Comment (enricostano) [inline app/views/shared/_movements.html.haml]: `%strong{ class: mv.amount.positive? ? "text-success" : "text-danger" }`
te he quitado 3 caracteres mas :joy:

Comment (enricostano) [inline app/views/shared/_movements.html.haml]: BTW... lo has probado? https://travis-ci.org/coopdevs/timeoverflow/builds/126176717#L429

esto falla por lo de `positive?`

Comment (sseerrggii) [inline app/views/shared/_movements.html.haml]: A mi me va

Comment (enricostano) [inline app/views/shared/_movements.html.haml]: `positive?` is available in ruby `2.3.0` but not in `2.2.2` which is the version that we use in travis.

https://github.com/coopdevs/timeoverflow/pull/206 fix this

grande @sseerrggii :clap:

**Comment by rewritten** [comment]
In a helper:

```ruby
TEXT_SUCCESS = 'text-success'.freeze
TEXT_DANGER = 'text-danger'.freeze

def green_red(amount)
  case amount <=> 0
  when -1 then TEXT_DANGER
  when 0 then nil
  when 1 then TEXT_SUCCESS
  end
end
```

And in the view:

```haml
%strong.lead{class: green_red(@member.account&.balance)
```

Note the use of the safe dot (`&.`) instead of try, again as a nice gift from Matz in ruby 2.3

**Comment by enricostano** [comment]
👍

**Pull request merged**